### DISCUSSION TITLE
Thai translation

### DISCUSSION THREAD

**Discussion by anchan42** [vuejs-translations/guidelines#15] [General]
OMG, I was halfway through the old document version (about 20% to be exact😆)
Now I have to start again..but how?

**Comment by goffxnca** [comment]
Maybe we can start it from here. https://github.com/goffxnca/vue3-doc-thai

**Reply by anchan42** [reply]
Yes, I got my own repo in https://github.com/anchan42/Vue-3-Thai-docs. The Essentials are done. The components in depth is about 80% done.

But that's the old version of the documents. Looks like there are still a lot more updates to come. Maybe we should wait until it is a bit more stable?

**Reply by goffxnca** [reply]
Wow! your translation looks awesome. yeah you're right we should wait a bit. pls let's me know bro when everything ready I'd love to join.

**Reply by anchan42** [reply]
Thanks, I am sure there are a few things can be improved. I am always struggling between transliteration or actual translation. Never felt that I got the balance right. 😕

**Reply by goffxnca** [reply]
I think it's might take a while before Evan starts creating the clear guideline for Vue3's translation. In the meantime, will you continue the rest for V.2 doc that you are currently working on? IMH there's quite a lot of Thai dev using Vue2 like y' know migrating to Vue3 esp. a large code-base is quite tricky. I would be more thrilled to join the V2 translation. as far as I know, there's also yet another v.2.7 to come before the end of Vue2 series. I just want to have an insight into what features already exists on v.2 and what's new in V.3 :D

**Comment by anchan42** [comment]
The one I started was for Vue 3. My last commit was more than a month ago, and I thought a lot of changes happened during that time. On second look, I am just 9 commits behind. I will continue the translation, but it's probably going to be awhile. You are welcome to join in. 😊

I am quite new to all of this, so you might need to tell me what to do to let you join in.
